src/_designs/components/labelType/index.vue

```diff
@@ -31,10 +31,11 @@ export default {
   @import "~@aeternity/aepp-components-3/src/styles/variables/colors";
   .label-wrapper {
```

Comment:
as i commented in #433 we should reuse the name of the component for the root element in the component. So `.label-type` instead of `.label-wrapper` I see now that you are using this generally.

Reply:
done